File: Difficulty: Easy/Max Sum Subarray of size K/max-sum-subarray-of-size-k.cpp
class Solution {
  public:
    int maxSubarraySum(vector<int>& arr, int k) {
        // code here
        int n=arr.size(),st=0,sum=0,ans=0;
        for(int i=0;i<n;i++){
            sum+=arr[i];
            if(i-st+1==k){
                ans=max(ans,sum);
                sum-=arr[st];
                st++;
            }
        }
        return ans;
    }
};

File: Difficulty: Easy/Max Sum Subarray of size K/max-sum-subarray-of-size-k.py
class Solution:
    def maxSubarraySum(self, arr, k):
        sum,st,ans=0,0,0
        
        for i in range(len(arr)):
            sum+=arr[i]
            ans=max(ans,sum)
            if (i-st+1)==k:
                sum-=arr[st]
                st+=1
                
        return ans        
                
        